mne/viz/_brain/_scraper.py

```diff
@@ -44,7 +45,13 @@ def __call__(self, block, block_vars, gallery_conf):
                             else:
                                 break
                         assert line.startswith("(") and line.endswith(")")
-                        kwargs.update(eval(f"dict{line}"))
+                        # Parse a=1, b='whatever', c=...
+                        # using literal_eval because it's safer
+                        line = line[1:-1]
+                        for item in line.split(","):
+                            item = item.strip()
+                            key, val = item.split("=", maxsplit=1)
+                            kwargs[key] = literal_eval(val)
```

Comment:
@drammock this was the key... `literal_eval` can't handle `dict(...)`, it needs to be `{'a': 1, ...` or so. I thought about using `re.sub` but that would rely on substituting `=`. The current code relies on substituting/splitting on `,` so if we ever use `,` in `# brain.save_movie(...)` magic comments in our code we'll need to change this.

Typing this out I'm tempted to change this back to `eval`. This parses comments only in our tutorials/examples. And if people want to execute malicious code there... they can just do it directly in the code. So I'm going to revert this and `# noqa` it